src/screens/sidebar/SideBar.tsx

```diff
@@ -167,7 +167,11 @@ export default class SideBar extends BaseScreen<Props, State> {
               <Text style={style.linkText}>{I18n.t('sidebar.statistics')}</Text>
             </ListItem>
             {this.securityProvider?.canListUsers() && (
-              <ListItem style={style.links} button iconLeft onPress={() => this.navigateTo('UsersNavigator', 'Users')}>
+              <ListItem
```

Comment:
Let the formatting to one line for short props (style={style.links}, button, iconLeft...)